---a/net/minecraft/world/storage/IPlayerFileData.java
+++b/net/minecraft/world/storage/IPlayerFileData.java
@@ -3,12 +3,12 @@
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.nbt.NBTTagCompound;
 
-public interface IPlayerFileData
-{
+public interface IPlayerFileData {
+
     void writePlayerData(EntityPlayer p_75753_1_);
 
     NBTTagCompound readPlayerData(EntityPlayer p_75752_1_);
 
     String[] getAvailablePlayerDat();
-}
+}
